lib/slather/coverage_service/coveralls.rb

```diff
@@ -32,6 +53,19 @@ def coveralls_coverage_data
           else
             raise StandardError, "Environment variable `TRAVIS_JOB_ID` not set. Is this running on a travis build?"
           end
+        elsif ci_service == :circleci
+          if circleci_job_id
```

Comment:
@ayanonagon we should probably look into moving these specific cases out into their own methods to cut down on clutter